Remove redefinitions of CPU types that cause warnings and mask that some
platform isn't actually supported on a host version.

diff -ur ld64-241.9/src/abstraction/MachOFileAbstraction.hpp work/ld64-241.9/src/abstraction/MachOFileAbstraction.hpp
--- ld64-241.9/src/abstraction/MachOFileAbstraction.hpp	2014-11-04 00:30:51.000000000 +0100
+++ ld64-241.9/src/abstraction/MachOFileAbstraction.hpp	2015-01-22 21:31:58.000000000 +0100
@@ -253,10 +253,6 @@
 
 
 
-// hack until arm64 headers are worked out
-#define CPU_TYPE_ARM64			(CPU_TYPE_ARM | CPU_ARCH_ABI64)
-#define CPU_SUBTYPE_ARM64_ALL	0
-#define CPU_SUBTYPE_ARM64_V8    1
 
 #define ARM64_RELOC_UNSIGNED            0 // for pointers
 #define ARM64_RELOC_SUBTRACTOR          1 // must be followed by a ARM64_RELOC_UNSIGNED
